part2/stages/Configure-GPT

```diff
@@ -42,34 +42,34 @@ mbr_in_whitelist()
 
 not_previous || exit ${Previous}
 
-if answerfile_specifies "install-mbr" ; then
-    INSTALL_MBR=$(read_xml_tag "${ANSWERFILE}" "install-mbr")
+if answerfile_specifies "install-gpt" ; then
+    INSTALL_GPT=$(read_xml_tag "${ANSWERFILE}" "install-gpt")
 
-    case "${INSTALL_MBR}" in
+    case "${INSTALL_GPT}" in
         true|false|auto)
             ;;
 
         "")
-            INSTALL_MBR="auto"
+            INSTALL_GPT="auto"
             ;;
 
         *)
-            echo "ERROR: Answerfile specifies unknown value '${INSTALL_MBR}'" \
-                 "for <install-mbr> tag." >&2
+            echo "ERROR: Answerfile specifies unknown value '${INSTALL_GPT}'" \
+                 "for <install-gpt> tag." >&2
             exit ${Abort}
     esac
 else
-    INSTALL_MBR="auto"
+    INSTALL_GPT="auto"
 fi
 
-if [ "${INSTALL_MBR}" = "auto" ] ; then
+if [ "${INSTALL_GPT}" = "auto" ] ; then
     if mbr_in_whitelist ; then
```

Comment:
Should mbr_in_whitelist be removed with the switch to GPT?

Reply:
Possibly, I'm not sure what `mbr_in_whitelist`'s function was but certainly doesn't sound like it's needed anymore. cc @crogers1